python/lsst/rubintv/production/plotting/latissNightReportPlots.py

```diff
@@ -0,0 +1,1000 @@
+        # TODO: generalise this to all bands and add checks for if empty
+        rband = np.where(bands == 'SDSSr_65mm')
+        gband = np.where(bands == 'SDSSg_65mm')
+        iband = np.where(bands == 'SDSSi_65mm')
+        zeroPoint = np.array(metadata['Zeropoint'][inds])
+        plt.plot(rawDates[gband], zeroPoint[gband], '.', color=gcolor, linestyle='-', label='SDSSg')
+        plt.plot(rawDates[rband], zeroPoint[rband], '.', color=rcolor, linestyle='-', label='SDSSr')
+        plt.plot(rawDates[iband], zeroPoint[iband], '.', color=icolor, linestyle='-', label='SDSSi')
+        plt.xlabel('TAI Date')
+        plt.ylabel('Photometric Zeropoint (mag)')
+        plt.xticks(rotation=25, horizontalalignment='right')
+        plt.grid()
+        ax = plt.gca()
+        xfmt = md.DateFormatter('%m-%d %H:%M:%S')
+        ax.xaxis.set_major_formatter(xfmt)
+        ax.tick_params(which='both', direction='in')
+        ax.minorticks_on()
+        plt.legend()
+        return True
+
+
+class SkyMeanPlot(LatissPlot):
+    _PlotName = 'Per-Band-Sky-Mean'
+    _PlotGroup = 'Photometry'
+
+    def __init__(self,
+                 dayObs,
+                 locationConfig=None,
+                 uploader=None):
+        super().__init__(dayObs=dayObs,
+                         plotName=self._PlotName,
+                         plotGroup=self._PlotGroup,
+                         locationConfig=locationConfig,
+                         uploader=uploader)
+
+    def plot(self, nightReport, metadata, ccdVisitTable):
+        """Create the zeropoint plot.
+
+        Parameters
+        ----------
+        nightReport : `lsst.rubintv.production.nightReport.NightReport`
+            The night report for the current night.
+        metadata : `pandas.DataFrame`
+            The front page metadata, as a dataframe.
+        ccdVisitTable : `pandas.DataFrame`
+            The visit summary table for the current day.
+
+        Returns
+        -------
+        success : `bool`
+            Did the plotting succeed, and thus upload should be performed?
+        """
+        for item in ['Sky mean', 'Filter']:
+            if item not in metadata.columns:
+                msg = f'Cannot create {self._PlotName} plot as required item {item} is not in the table.'
+                self.log.warning(msg)
+                return False
+
+        # TODO: get a figure you can reuse to avoid matplotlib memory leak
+        plt.figure(constrained_layout=True)
+
+        datesDict = nightReport.getDatesForSeqNums()
+
+        inds = metadata.index[metadata['Sky mean'] > 0].tolist()  # get the non-nan values
+        rawDates = np.asarray([datesDict[seqNum] for seqNum in inds])
+        bands = np.asarray(metadata['Filter'][inds])
+        # TODO: generalise this to all bands and add checks for if empty
+        rband = np.where(bands == 'SDSSr_65mm')
+        gband = np.where(bands == 'SDSSg_65mm')
+        iband = np.where(bands == 'SDSSi_65mm')
```

Comment:
It'd be good to add a little indirection here that lets you say `r`, `i`, or `z` here by looking up the full expected string in some kind of mapping, even if that's just a constant at the top of the file.

Reply:
This definitely needs doing, but I'll file a new ticket for it.

Reply:
https://jira.lsstcorp.org/browse/DM-38287 and I've added a TODO to the code with that.